libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json

```diff
@@ -235,11 +240,41 @@
         "comment": "Attachment of the scheduled email.",
         "limit": 0
     },
+    "dialogs.schedule.email.attachments.label": {
+        "value": "Attachments:",
+        "comment": "Attachments of the scheduled email.",
+        "limit": 0
+    },
+    "dialogs.schedule.email.attachment.select": {
+        "value": "Select attachments",
+        "comment": "Label of a button that opens attachment selection",
+        "limit": 0
+    },
+    "dialogs.schedule.email.attachment.format": {
+        "value": "Format options",
+        "comment": "Label of a button that opens format configuration",
+        "limit": 0
+    },
+    "dialogs.schedule.email.attachment.select.dashboard.header": {
+        "value": "Dashboard (PDF)",
+        "comment": "Heading of checkbox that selects dashboard to be exported in PDF file format",
+        "limit": 0
+    },
+    "dialogs.schedule.email.attachment.select.widgets.header": {
+        "value": "Insights (CSV, XLSX)",
+        "comment": "Heading of section of checkboxes for selecting insights to be exported as CSV of XLSX file",
+        "limit": 0
+    },
```

Comment:
Missing translation for report. Also check other translations regarding this please

Reply:
good catch!